src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -745,6 +806,8 @@ def _upload_objects_in_parallel(
                                                          workspace_id,
                                                          load_id,
                                                          obj_tuples,
+                                                         asu_client,
```

Comment:
What happens if an assembly is uploaded but the genome upload doesn't complete?

Reply:
it first does `_query_workspace_with_load_id_mass` to check successfully uploaded Genomes. If Genome hasn't created, it will try to create Genome in the second run.

Reply:
What if there is an assembly though?

Reply:
`_query_workspace_with_load_id_mass` won't find previous uploaded Assemblies since the name will be different. Genome obj name is exactly the same as genbank file name.

Reply:
Ok, so if an assembly gets uploaded but it's genome doesn't, the assembly will be duplicated next load?

Reply:
yup. The assembly will be duplicated during the next load.

Reply:
Ok, I guess that's ok for now.